lib/Fetcher/FeedFetcher.php

```diff
@@ -325,6 +325,8 @@ protected function getFavicon(FeedInterface $feed, string $url)
     {
         $favicon = $feed->getLogo();
 
+        ini_set('user_agent', 'NextCloud-News/1.0');
+
```

Comment:
I think we could also just use Guzzle to download the file, but I'm fine trying this first.